.coveragerc

```diff
@@ -4,10 +4,8 @@ source = gocept.sftpcopy
 
 [report]
 precision = 2
-omit = */interfaces.py
-       */interfaces
-       */tests/*
-       */tests.py
+omit =
+    pragma: no-cover
```

Comment:
The option for exlcuding lines is 
```
exclude_lines =
    pragma: nocover
```
But I guess we can leave out the omit option completely here, as I did not even find `pragma` somwhere in the code.

Reply:
Thank you, I forgot to save the file I was using `nocover`. I am going to add some more.